core/cap-0045.md

```diff
@@ -311,91 +320,77 @@ then we should acknowledge that no prices at all is probably just as useful.
+
+For an Arrow-Debreu market, the budget-constraint must also hold:
+
+<div align="center"><img style="background: white;" src="https://render.githubusercontent.com/render/math?math=pX%2BY%20%3D%20pX'%2BY'"></div>
+
+
+Solving the budget constraint and marginal price constraint together, we get the demand for `X` and `Y` when <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=p%20%3C%20%5Cfrac%7B(1-F)Y%7D%7BX%7D">
+:
+<div align="center"><img style="background: white;" src="https://render.githubusercontent.com/render/math?math=X'%20%3D%20%5Cfrac%7B(pX%2BY)(1-F)%7D%7Bp%5Ccdot(2-F)%7D"></div>
+
+<div align="center"><img style="background: white;" src="https://render.githubusercontent.com/render/math?math=Y'%3D%5Cfrac%7BpX%2BY%7D%7B2-F%7D"></div>
+
+Observe that in this case, <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=X'%20%3D%20%5Cfrac%7B(pX%2BY)(1-F)%7D%7Bp%5Ccdot(2-F)%7D%20%3E%20%5Cfrac%7BpX(1-F)%2BpX%7D%7Bp%5Ccdot(2-F)%7D%20%3D%20X">
+
+
+and <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=Y'%3D%5Cfrac%7BpX%2BY%7D%7B2-F%7D%3C%5Cfrac%7B(1-F)Y%2BY%7D%7B2-F%7D%3DY">
+
+Thus, the liquidity pool, as expected, sells `Y` and purchases `X`.
+
+By the exact same notion, when <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=%5Cfrac%7B1%7D%7Bp%7D%3C%5Cfrac%7B(1-F)X%7D%7BY%7D">, the demand for `X` and `Y` are:
+<div align="center"><img style="background: white;" src="https://render.githubusercontent.com/render/math?math=X'%3D%5Cfrac%7BpX%2BY%7D%7B(2-F)%5Ccdot%20p%7D"></div>
+<div align="center"><img style="background: white;" src="https://render.githubusercontent.com/render/math?math=Y'%3D%5Cfrac%7B(pX%2BY)(1-F)%7D%7B2-F%7D"></div>
+
+Observe in this case, <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=X'%3D%5Cfrac%7BpX%2BY%7D%7B(2-F)%5Ccdot%20p%7D%3C%5Cfrac%7BpX%2BpX(1-F)%7D%7B(2-F)%5Ccdot%20p%7D%3DX"> and <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=Y'%3D%5Cfrac%7B(pX%2BY)(1-F)%7D%7B2-F%7D%3E%5Cfrac%7B(Y%2B(1-F)Y)%7D%7B2-F%7D%20%3DY">
 
 Thus, the liquidity pool, as expected, sells `X` and purchases `Y`.
 
+When the price is in between, i.e. <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=%5Cfrac%7B(1-F)Y%7D%7BX%7D%20%5Cle%20p%20%5Cle%20%5Cfrac%7BY%7D%7B(1-F)X%7D">, no trade will happen and both reserves stay the same.
+
 ### Demand from an order
 Tatonnement does not handle discontinuities well. The demand from an order is
 intrinsically discontinuous, so a natural solution is to smooth the demand. This
```

Comment:
There was some confusion about what should be smoothed. I think you should clarify that the price-weighted demand should be linearly interpolated.

Reply:
I clarified it. Also, I thought the full illustration with math should've remove any confusion.